md/cache-configuration-and-policy.md

```diff
@@ -1,16 +1,31 @@
-# Cache configuration and policy
+## Cache configuration and policy 
 
-This page explain how the portal cache and the living application cache are configured in bonita. You will also find 
+This page explain how the portal cache and the living application cache are configured in bonita 7.6 version. You will also find 
 details on the impact this cache configuration can have for the end user, and on how you can modify this configuration.
 
```

Comment:
maybe had an cache policy over view section :

"
#### Cache policy over view
The default cache policy configured on your bonita bundle will allow your browser to stores recently used information so that it can be quickly accessed at a later time, thereby improving performance.
When you browse through different forms or custom pages that use the same images, CSS, or JavaScript, your browser will not have to re-download the files. Instead, the browser can simply load them from the cache, which is stored on your local hard drive.

This cache policy will have impacts at three different levels:
- Bonita portal display time and look and feel update.
- Living applications display time and look and feel update.
- Mobile application display time and look and feel update.
"